CompanyEmployees.Core.Domain/Entities/Employee.cs

```diff
@@ -22,5 +22,8 @@ public class Employee
         [ForeignKey(nameof(Company))]
         public Guid CompanyId { get; set; }
```

Comment:
Avoid using public fields directly. Replace this with a property using PascalCase (`Salary`) to conform to .NET naming conventions and encapsulation best practices. Also, strongly consider using an appropriate numeric type (e.g., `decimal`) instead of a `string` to represent monetary values.

```suggestion
public decimal Salary { get; set; }
```